app/src/main/java/org/inspire/breath/validateDiarrhoea.java

```diff
@@ -0,0 +1,59 @@
+package org.inspire.breath;
+
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.TextView;
+import android.widget.Button;
+
+public class validateDiarrhoea extends AppCompatActivity {
```

Comment:
could you move this class into org.inspire.breath.activities (see master branch)?